### PR TITLE
Added the tasks about cucumber module by using POM 

### PR DESCRIPTION
**Pull request by munirsaracoglu**
Added the tasks about cucumber module by using POM and there are four files created 1.sample_scenario.feature 2.sample_steps.rb 3.to_do_list.rb 4.hooks.rb

### CODE REVIEW

File: backend/features/pages/to_do_list.rb [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  def item_displayed?(item)
+    page.has_text?(item)
+  end
+
+  def click_checkbox_near_item
+    find(:css, "input[type='checkbox']").click
+  end
+
+  def delete_this_text_displayed?
+    page.has_xpath?("//span[@class='DeleteItem']", wait: 5, visible: true)
```
Comment (danylo-soroka) [inline backend/features/pages/to_do_list.rb]: I see we are waiting here for 5 seconds before the delete button is visible. Let's try to improve the actual application which will also help to make our test faster and more stable:
There is a file inside `backend` directory that contains all the back-end functionality. We need to find a method/endpoint which is responsible for completing the tasks and to remove one unnecessary line (hint: it obligates the code to wait from 2 to 5 before proceed)

File: backend/features/sample_scenario.feature [hunk 2]
```diff
@@ -1,6 +1,27 @@
...
     Given I am in ToDo Lists app
-    Then I should see a button with text "Add"
+
+  Scenario Outline: Add some lists to the to-do list
+    When I add a "<list>" to the to-do list
+    Then I should see the "<list>" added to the list
+    And I should see the number of lists is not equal to 0
+
+    Examples:
+      |list|
```
Comment (danylo-soroka) [inline backend/features/sample_scenario.feature]: JFYI, there is a [shortcut](https://www.jetbrains.com/help/ruby/reformat-and-rearrange-code.html#reformat_code) to reformat the code in Rubymine. It helps to keep tables and their borders aligned

Comment (munirsaracoglu) [reply backend/features/sample_scenario.feature]: it is done

**Review by danylo-soroka — APPROVED** [review]
Amazing! Great job 💪

**Pull request merged**